src/ui/Overlay.js

```diff
@@ -1,30 +1,44 @@
 import { Suspense, useEffect, useState } from 'react'
 import { Footer } from '@pmndrs/branding'
+import { useProgress } from '@react-three/drei'
 import { Keys } from './Help'
 import { useStore } from '../store'
 
-function Ready({ setReady }) {
-  useEffect(() => () => void setReady(true), [])
-  return null
+function Ready() {
+  const { progress } = useProgress()
```

Comment:
We want ready to be set as true when progress is complete

```suggestion
const { progress } = useProgress()
useEffect(() => {
   if(progress !== 100) { return }
   setReady(true)
}, [progress])
```

Reply:
So I added setReady back along with the other stuff that conditionally rendered based on `ready`. I'm curious as to why the ready state is preferred since it seems like the loading screen will never be triggered.